policyengine_canada/parameters/gov/provinces/mb/tax/income/credits/tuition_amount/amount/part_time/disabled.yaml

```diff
@@ -0,0 +1,11 @@
+description: Manitoba extends this tuition amount to disabled part-time students.
+values:
+  2022-01-01: 4_800 # 400/month * 12
+metadata:
+  unit: currency-CAD
+  label: Manitoba tuition credit disabled part-time student amount
+  reference:
+    - title: Government of Canada - Manitoba Information Guide
+      href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1mb/td1mb-23e.pdf#page=1
+    - title: Government of Canada - Manitoba individual income tax C.C.S.M C.l.10 Division II Section4.6(14)(B)(a)
+      href: https://web2.gov.mb.ca/laws/statutes/archive/i010(2020-11-05)e.php#4.6(14)
```

Comment:
I only see one $400 value for part-time disabled and full-time, and it's the same citation. Is it only defined once? I didn't see it in the cited https://laws-lois.justice.gc.ca/eng/acts/I-3.3/section-118.6.html

If it's just one number in the legal code, let's only represent it as a single parameter.